config/docker/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# NAME:     bec/participa
+# VERSION:  1.0.0
+
+FROM buildpack-deps:xenial
+
+LABEL \
+  cat.bcnencomu.participa.description="Docker image to run BeC's Participa platform" \
+  cat.bcnencomu.participa.vendor="BeC" \
+  cat.bcnencomu.participa.version="1.0.0"
+
+MAINTAINER Carles Muiños "carles@adabits.org"
+
+# Base
+## add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN useradd participa -s /bin/bash -m -U
+
+RUN \
+  apt-get update && apt-get install -y --no-install-recommends \
+  build-essential \
+  ghostscript \
+  libicu55 \
+  nodejs \
+  sqlite3 \
+  wkhtmltopdf \
+&& rm -rf /var/lib/apt/lists/*
+
+# Ruby
+ENV RUBY_VERSION 2.2.2
+ENV RUBY_HOME /usr/local/ruby
+RUN \
+  set -ex \
+  && mkdir -p /usr/local/etc \
+  && { \
+      echo 'install: --no-document'; \
+      echo 'update: --no-document'; \
+     } >> /usr/local/etc/gemrc \
+  && RUBY_SRC="$(mktemp -d)" \
+  && git clone https://github.com/rbenv/ruby-build.git $RUBY_SRC \
+  && $RUBY_SRC/install.sh \
+  && rm -rf $RUBY_SRC \
+  && mkdir -p $RUBY_HOME \
+  && ruby-build $RUBY_VERSION $RUBY_HOME/$RUBY_VERSION \
+  && ln -s $RUBY_HOME/$RUBY_VERSION $RUBY_HOME/default
+
+ENV PATH $RUBY_HOME/default/bin:$PATH
+RUN gem install bundler --no-document
+
+# PhantomJS
+ENV PHANTOMJS_VERSION 2.1.1
+RUN \
+  set -ex \
+  && wget -q -O phantomjs.tar.bz2 https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-${PHANTOMJS_VERSION}-linux-x86_64.tar.bz2 \
+  && mkdir -p /usr/local/share/phantomjs \
+  && tar -xjf phantomjs.tar.bz2 -C /usr/local/share/phantomjs --strip-components=1 \
+  && rm phantomjs.tar.bz2 \
+  && ln -s /usr/local/share/phantomjs/bin/phantomjs /usr/local/bin/phantomjs
```

Comment:
`phantomjs 2.1.1` is available in `xenial`

```
$ apt policy phantomjs
phantomjs:
  Instalados: 2.1.1+dfsg-1
  Candidato:  2.1.1+dfsg-1
  Tabla de versión:
 *** 2.1.1+dfsg-1 500
        500 http://ubuntu.c3sl.ufpr.br/ubuntu xenial/universe amd64 Packages
        100 /var/lib/dpkg/status
```

So all that could be simplified, no? Or is there any reason to install it like that?